app/templates/admin-panel/global-system-message.hbs

```diff
@@ -0,0 +1,25 @@
+<AuToolbar @size="large" @nowrap="true" as |Group|>
+  <Group>
+    <AuHeading @skin="2">System message</AuHeading>
+  </Group>
+</AuToolbar>
+<div class="au-c-body-container au-c-body-container--scroll">
+  <div class="au-o-box au-u-flex au-u-flex--column">
+    <span class="au-u-margin-bottom">
+      <AuLabel for="global-system-message-input">
+        Message
+      </AuLabel>
+      <AuInput
+        id="global-system-message-input"
+        value={{this.message}}
```

Comment:
```suggestion
        value={{this.message}}
        @width="block"
```

![image](https://github.com/user-attachments/assets/fd15787d-0b12-409b-9296-c70df24a9851)

Reply:
Also added `rows={{2}}` so it is a little bigger when your text is somewhat long